test/unit/test_Rationals.cpp

```diff
@@ -132,16 +132,6 @@ TEST(Rationals_test, test_overwrite)
     r = q;
 }
 
-TEST(Rationals_test, test_unique)
-{
-    vec<Real> v = {1, 1, 2, 3, 3, 4};
-    vec<Real> w = {1, 2, 3, 4};
-    uniq(v);
-    ASSERT_EQ(v.size(), w.size());
-    for (int i = 0; i < v.size(); i++)
-        ASSERT_EQ(v[i], w[i]);
-}
-
```

Comment:
Should we replace this test with something else?

Reply:
I added a unit test for FastRational::get_multiplicand, that's how it should have been in the first place I think.

Reply:
Maybe also add a test that would exercise the use of unique in the implementation.
Maybe with values `1/3, 1/5, 2/5`?

Reply:
I added another unit test here, as you suggested below.